pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "smile-id-core"
-version = "1.0.9"
+version = "1.1.0"
```

this needs to be a 2.0.0 because of the breaking API change, but don't change it until we issue a release. 
```suggestion
version = "1.0.9"
```